.github/workflows/push_tests.yml

```diff
@@ -19,10 +19,10 @@ jobs:
       - uses: actions/checkout@v2
       - run: |
           git fetch --depth=1 origin +${{github.base_ref}}
-      - name: Set up Python 3.7
+      - name: Set up Python 3.8
```

Comment:
good reason not to upgrade higher?

Reply:
Yes exactly! matching docker python version.